app/views/welcome/_search.html.haml

```diff
@@ -0,0 +1,10 @@
+= search_form_for @q do |f|
+  .form-group
+    .input-group
+      = f.search_field :name_cont, class: 'form-control', placeholder: 'Search by community name'
+
+      .input-group-btn
+        = link_to communities_path, class: 'btn btn-default', data: { behavior: 'clear-field', target: '#q_name_cont', autosubmit: 'true' }  do
```

Comment:
I wouldn't even include this button as the javascript to handle this doesn't exist in this project and isn't really necessary from the welcome view of the system.